### PR TITLE
changes by krishna

### PR DESCRIPTION
**Pull request by saurabhj0709**
Added Cart.Js file and changes in app.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import Header from './components/Layout/Header';
 import Meals from './components/Meals/Meals';
 import CartProvider from './store/CartProvider';
+import Cart from './components/Cart/Cart';
 
 function App() {
   return (
     <CartProvider>
+      <Cart />
```
Comment (saurabhj0709) [inline src/App.js]: Delete this line

**Review by saurabhj0709 — COMMENTED** [review]
Please finish this

File: src/App.js (hunk 1 above)
Comment (1-krishna) [reply src/App.js]: delete kar denege bhai

**Comment by saurabhj0709** [comment]
Ni Chahiye

**Pull request closed without merging**